framework/src/meshmodifiers/SidesetAroundSubdomainUpdater.C

```diff
@@ -165,6 +166,15 @@ SidesetAroundSubdomainUpdater::finalize()
   const auto & mesh = _mesh.getMesh();
   const auto * displaced_mesh =
       _displaced_problem ? &_displaced_problem->mesh().getMesh() : nullptr;
+  if (getParam<bool>("verbose"))
+  {
+    if (_remove.size())
+      _console << "Removing " << _remove.size() << " sides from sideset " << _boundary_id
+               << " on processor " << _pid << std::endl;
+    if (_add.size())
+      _console << "Adding " << _add.size() << " sides to sideset " << _boundary_id
+               << " on processor " << _pid << std::endl;
```

Comment:
I don't have a great suggestion for what to change this to, but the current language makes it seem like the boundary information will be out of sync across processes, e.g. we're only adding/removing it on `_pid` but in truth we're going to do a global communication to make sure we do it in sync across processes. Honestly might be best just to remove the part including the `_pid`